.gitignore

```diff
@@ -1,3 +1,6 @@
-.DS_Store
-.idea
-/vendor
+*
```

Comment:
Hmm, I'm not big fan of astralisk. why it's needed?

Reply:
* - ignor all files 
! - exclude from ignor
I noticed that in the Magento we have no /vendor folder and all Botl plugin's folders have name "Bolt", "bolt', "Boltpay" and "bolpay"
Such gitignore file allow to work with git when plugin is integrated
Does it make sense?

Reply:
I know what astarisk means :) it sounds like you put plugin code within magento directory? you can avoid that by using modman - https://github.com/BoltApp/bolt-magento1#modman-guide

Reply:
@oldDriver You have missed and have not been informed that we have already thoroughly focused on this problem and evolved to the modman solution.

Still, I'll give you some feedback on the solution that you presented.  The ignore all (*) except for  double asterisk directory approach will several critical files that do not lie in a sub-directory of something that says `bolt` or `boltpay`, (e.g. layout XML files, top level .md files, .sh files etc.) 

Compare it to the PR that replaced our previous asterisk solution:
https://github.com/BoltApp/bolt-magento1/pull/155/files

The problem with the previous solution was that it was ignoring new files and people were often forgetting to add the new files in their commits.  This PR will re-introduce that problem.